webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpParallelCoordinates.tsx

```diff
@@ -225,6 +227,10 @@ const HpParallelCoordinates: React.FC<Props> = ({
 
   const clearSelected = useCallback(() => setSelectedRowKeys([]), []);
 
+  useEffect(() => {
+    if (showCompareTrials) trialsComparisonModal.open();
+  }, [showCompareTrials, trialsComparisonModal]);
```

Comment:
All of these sections with a `showCompareTrials` / `setShowCompareTrials` state can now be replaced by calling `trialsComparisonModal.open()`